numpy/core/include/numpy/npy_cpu.h

```diff
@@ -39,6 +39,12 @@
      * _M_AMD64 defined by MS compiler
      */
     #define NPY_CPU_AMD64
+#elif defined(_M_ARM64)
+    /*
+     * _M_ARM64 is defined by VS compiler >= 2017 to build native
+     * ARM64 packages on Windows
+     */
+    #define NPY_CPU_ARMEL_AARCH64
```

Comment:
why `ARMEL_AARCH64` ? and not `MSVC_AARCH64`?

Reply:
As this is for endianess check, the closest was armel_aarch64.